.travis.yml

```diff
@@ -77,6 +77,9 @@ jobs:
         - env: ENABLE_ZTS=1 ENABLE_DEBUG=1 SKIP_IO_CAPTURE_TESTS=1 S390X=1
           arch: s390x
 
+before_install:
+    - sudo locale-gen de_DE en_US fi_FI fr_FR fr_FR@euro hr_HR is_IS it_IT pt_PT ru_RU.KOI8-R sv_SE
```

Comment:
It would be nice to normalize the locale usage, so it's possible to run tests with a minimal set. For example, we should be using only de_DE to check for issues related to comma as decimal separator, while currently probably fi_FI, fr_FR, etc are used for the same purpose, simply depending on which locale a bug was reported against.

Reply:
I agree. Can we merge to get the tests passing and I'll try to circle back and prune some of those locales.